### PR TITLE
excel_inspector

### PR DESCRIPTION
**Pull request by Vilma-Agalioti**
# Pull Request Template

**script name** - excel_inspector.py

**Brief about script**
It loops through .xlsx files of a folder, it opens them, parses their sheets and prints: filename of each spreadsheet, sheets included, basic info about the sheets.

### Self Check(Tick After Making pull Request)

- [ ] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in proper folder (Name of folder should be in lowercase with no space in between)
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [x] I have added README.md and requirements.txt with my script

If issue was not assigned to you Please don't make a PR. It will marked as invalid.

### CODE REVIEW

File: excel_inspector/excel_inspector.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+for path, subdirs, files in os.walk(sheets_folder):
+    for filename in files:
+        print('\n [] File:', filename, '===============')
+        if filename.endswith('.xlsx'):
+            excel = pd.ExcelFile(path + '\\' + filename)
+            print('Number of sheets:', len(excel.sheet_names))
+            print('Sheet names:', excel.sheet_names)
+            for sheet in excel.sheet_names:
+                df = excel.parse(sheet)
+                print('Sheet:', sheet, ' with the columns:', list(df.columns) )
```
Comment (thevirtualbuddy) [inline excel_inspector/excel_inspector.py:16]: Remove the whitespace before ')' 
`print('Sheet:', sheet, ' with the columns:', list(df.columns))`

File: excel_inspector/excel_inspector.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Parse spreadsheets and its sheets ===========================================
+import pandas as pd
+import os
+
+sheets_folder = r'' # add your folder path
```
Comment (thevirtualbuddy) [inline excel_inspector/excel_inspector.py:5]: Add an extra space after '#'
`sheets_folder = r'' #  add your folder path`

**Review by thevirtualbuddy — COMMENTED** [review]
Removing that extra space before ')' in Line 16 and adding an extra space after the '#' in Line 5 would help to pass the Flake8Linter check successfully. 👍

**Comment by pawangeek** [comment]
@all-contributors add @Vilma-Agalioti for code

**Pull request merged**